docs/pages/blog/premium-plan-release.md

```diff
@@ -0,0 +1,146 @@
+
+- **Removing volume and renewal discounts**
+
+  We aim to give a more transparent price based on the number of developer seats while providing more predictable revenue for MUI X in the long term.
+  We believe that’s meaningful for both our long-standing customers, and MUI as a company.
+
+- **Perpetual in production**
+
+  Customers can use any licensed version perpetually in a production environment (deployed/published applications).
+  But an active license will always be required during development.
+  We roll bug fixes, performance enhancements and other improvements into new releases, so we want to encourage projects in active development to use a current version.
+
+- **Price cap for Premium plan**
+
+  The price for Pro is still capped at 10 seats, but we’re removing the concept of price cap for Premium. We kindly ask you to [contact sales](mailto:sales@mui.com) if you wish to purchase a license for more than 50 developers.
+
+Please, feel free to read the new [license agreement](https://mui.com/store/legal/mui-x-eula/) in detail.
+
+## What can you expect next?
+
+We mentioned the aggregation functions already,
+but there’s a [lot more coming](https://github.com/mui/mui-x/projects/1), and certainly not only for Premium users.
+We’ll continue to enhance all of the components and are
+putting more effort into usability and overall user and developer experience.
+A product designer is soon joining the team, and we aim to keep exploring the most advanced use cases for data-rich applications,
+while working on bringing the best UX and DX in the market.
+
+We’re very excited about the possibilities the new Premium features will enable, and we hope you’re excited too!
+
+## FAQ 🔧
+
+### How to migrate?
+
+To migrate to the premium version in your codebase, install the dedicated package `@mui/x-data-grid-premium` as follows:
+
+With npm:
+
+```js
+npm install @mui/x-data-grid-premium
+```
+
+With yarn:
+
+```js
+yarn add @mui/x-data-grid-premium
+```
+
+All the features from Community and Pro plan are available in the Premium package. You can remove your previous data grid package without any concern.
+
+The new package exports the component `<DataGridPremium />` which has basically the same API as `<DataGridPro />`. To use this package for development of a production product, and to remove the watermark, you must acquire a Premium license. If you wish to upgrade your existing Pro license, please [contact sales](mailto:sales@mui.com).
+
+### Should I update my code?
+
+Unless you want to upgrade to Premium, or if you’re already using row-grouping in your project, you don’t have to do anything.
+
+If you're using Row grouping with the Pro plan, it's now officially deprecated and will be removed from the experimental features [as planned](https://mui.com/blog/introducing-the-row-grouping-feature/#how-to-unlock-this-feature). To continue using it, please upgrade to the Premium plan.
+
+### What are the initial Excel export features?
```

Comment:
This question is kind of clunky. I would rephrase it to make it sound more like something a reader would ask.

```suggestion
### What can I do with the Excel export feature?
```